schema/pet-insurance.json

```diff
@@ -78,7 +77,11 @@
             "typeRef": "#/domains/PetInsurance/types/CoverOptions",
             "sourceOutputKey": "availableCoverOptions",
             "inputMethod": "SelectMany"
-
+        },
+        "selectedVetPaymentTerm": {

```

Comment:
needs `description`, what is it and how is it different from `selectedPaymentTerm`?